packages/Path-View-Tests.package/SClassDiagramTest.class/instance/testZoomingWithSlider.st

```diff
@@ -0,0 +1,12 @@
+zooming
+testZoomingWithSlider
+
+	| pathView slider diagram newZoomLvl |
+
+	pathView := SPathView privateNew buildWith: MorphicToolBuilder new.
+	slider := (pathView allSubmorphsOfKind: UiSlider) asOrderedCollection first.
+	diagram := (pathView allSubmorphsOfKind: SClassDiagram) asOrderedCollection first.
+	newZoomLvl := 0.5. "Should not be default value"
```

Comment:
The slider is set to "0.5". The test is successfully if the parent morph (ScrollPane) contains the same value